test/convenience-method-test.js

```diff
@@ -1,6 +1,7 @@
 var sinon = require('sinon');
 var main = require('../package.json').main;
 var schedule = require('../' + main);
+var {RRule} = require('rrule');
```

Comment:
Will this work with Node 0.10?

Reply:
Ill test this and get back to you, I am not aware of the limitations of Node 0.10

Reply:
I'm starting next semver major soon, so feel free to only support Node 6+